src/main/java/net/minecraftforge/client/model/pipeline/VertexLighterFlat.java

```diff
@@ -53,8 +53,9 @@ public VertexLighterFlat(BlockColors colors)
     public void setParent(IVertexConsumer parent)
     {
         super.setParent(parent);
-        if(Objects.equal(getVertexFormat(), parent.getVertexFormat())) return;
-        setVertexFormat(getVertexFormat(parent));
+        VertexFormat format = getVertexFormat(parent);
+        if(format.equals(getVertexFormat())) return;
```

Comment:
I can keep that, but, when looking at the code it seemed that `parent.getVertexFormat()` shouldn't return null. For instance, in that case, there'd just be an NPE later ([see here](https://github.com/MinecraftForge/MinecraftForge/blob/1.11.x/src/main/java/net/minecraftforge/client/model/pipeline/VertexLighterFlat.java#L144-L145)).